docs/compilers/CSharp/Deviations from Standard.md

```diff
@@ -35,3 +35,38 @@ class EnumConversionTest
 ```
 
 Conversions are (correctly) *not* permitted from constant expressions which have a type of `bool`, other enumerations, or reference types.
+
+# Member lookup
```

Comment:
Should this be second-level heading (`##`)? Similarly would fix the existing one.

Reply:
I think it was intentional to have "Implicit enum conversions from zero" under "Conversions".
But "Member lookup" doesn't belong under "Conversions", so I have it at top-level